CHANGELOG

```diff
@@ -1,6 +1,45 @@
 v3.7.7 (XXXX-XX-XX)
 -------------------
 
+* Improvements for logging. This adds the following startup options to arangod:
+
+- `--log.max-entry-length`: controls the maximum line length for individual
+    log messages that are written into normal logfiles by arangod (note: this
+    does not include audit log messages). 
+    Any log messages longer than the specified value will be truncated and the 
+    suffix '...' will be added to them. The purpose of this parameter is to 
+    shorten long log messages in case there is not a lot of space for logfiles, 
+    and to keep rogue log messages from overusing resources. 
+    The default value is 128 MB, which is very high and should effectively
+    mean downwards-compatiblity with previous arangod versions, which did not
+    restrict the maximum size of log messages.
+  
+  - `--audit.max-entry-length`: controls the maximum line length for individual
+    audit log messages that are written into audit logs by arangod. Any audit
+    log messages longer than the specified value will be truncated and the 
+    suffix '...' will be added to them. 
+    The default value is 128 MB, which is very high and should effectively
+    mean downwards-compatiblity with previous arangod versions, which did not
+    restrict the maximum size of log messages.
+
+  - `--log.in-memory-level`: controls which log messages are preserved in 
```

Comment:
Just a thought:
Imo we have different semantics for level 'info' (implying all levels below) and 'error', 'warning' (implying exactly that level).
Would it make sense to rename 'info' -> 'all'?

Reply:
Thanks, I fixed the description of the PR.
`info` is not equivalent to `all`, as there are also `debug` and `trace`, which are not mentioned in the description because they don't make so much sense here (the size of the in-memory buffer is limited, and when setting log level to debug or trace, it will very quickly be exceeded).